dbt/adapters/bigquery/connections.py

```diff
@@ -380,7 +380,7 @@ def execute(
             query_table = client.get_table(query_job.destination)
             code = 'CREATE TABLE'
             num_rows = query_table.num_rows
-            bytes_processed = query_job.total_bytes_processed
+            bytes_processed = query_job.total_bytes_processed or 0
```

Comment:
I think it would make more sense for `total_bytes_processed` to be `None` here, rather than `0`. Per its type, it _can_ be `None`:
https://github.com/dbt-labs/dbt-bigquery/blob/d6369963a25fdfc188b205c20bc4191bbe49faa1/dbt/adapters/bigquery/connections.py#L83

A few points:
- `format_bytes` + `format_rows_number` are only used in `dbt-bigquery`, so it would make sense to move those methods out of `dbt-core` (`core/dbt/utils.py`) and into this package/repo
- We should update the logic here, or in those methods, to return `None` if `bytes_processed` is `None`, rather than trying to wrap it in `abs()` and raising an error

Reply:
Hey @jtcohen6 I sign the CLA first with my personal account, than i figure out that my local github is linked with the company email, than I sign again with other email.

I make the change to be `0` to avoid change too much in the code and avoid to break in other part of the code, and it`s a safe approuch

Reply:
Maybe change to this?
```
def format_bytes(num_bytes):
    if num_bytes:
        for unit in ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB']:
            if abs(num_bytes) < 1024.0:
                return f"{num_bytes:3.1f} {unit}"
            num_bytes /= 1024.0

        num_bytes *= 1024.0
        return f"{num_bytes:3.1f} {unit}"
    
    else:
        return None
```
What do you think?

Reply:
That makes sense to me!

Reply:
I made a new change to add this methods to `bigquery/connections` so just need to be deleted from core.

Reply:
That sounds good to me. It's not a huge deal if that method remains unused in `core/utils.py` for the time being.